python/tests/test_highlevel.py

```diff
@@ -1807,7 +1807,11 @@ def test_table_metadata_round_trip_via_row_getters(self):
 
 
 def test_pickle_round_trip(ts_fixture):
-    assert ts_fixture.tables == pickle.loads(pickle.dumps(ts_fixture)).tables
+    for protocol in range(pickle.HIGHEST_PROTOCOL + 1):
+        ts = pickle.loads(pickle.dumps(ts_fixture, protocol=protocol))
+        assert ts.tables == ts_fixture.tables
+        # Do some thing to check the ts is init'd properly
+        ts.draw_text()
```

Comment:
Could be quite an expensive thing to do, maybe something like ``assert ts.num_trees == len(list(ts.trees()))``?

Reply:
That doesn't trigger the error unfortunately. The example ts is fixed and takes 40ms on my box:
```
0.04s call     tests/test_highlevel.py::test_pickle_round_trip
0.01s setup    tests/test_highlevel.py::test_pickle_round_trip
0.00s teardown tests/test_highlevel.py::test_pickle_round_trip
```